src/win.c

```diff
@@ -2305,23 +2306,15 @@ void map_win_start(session_t *ps, struct managed_win *w) {
 	log_debug("Window %#010x has opacity %f, opacity target is %f", w->base.id,
 	          w->opacity, w->opacity_target);
 
-	win_determine_blur_background(ps, w);
```

Comment:
this is called in `win_process_update_flags` -> `win_on_factor_change`

Reply:
Right, `WIN_FLAGS_FACTOR_CHANGED` is set in `fill_win` and has been triggered by the corresponding event else.